lib/Importer/Caffe2ModelLoader.cpp

```diff
@@ -343,7 +344,8 @@ llvm::Error Caffe2ModelLoader::loadConv(const caffe2::OperatorDef &op,
   // We expect the input to be NHWC.
   NodeValue finalIn;
   if (order == "NCHW") {
-    finalIn = G_.createTranspose(opName, in, NCHW2NHWC)->getResult();
+    finalIn = G_.createTranspose(opName, in, NCHW2NHWC, GLOW_DEFAULT_4D_LAYOUT)
```

Comment:
Please use a concrete string here, because any change to the definition of GLOW_DEFAULT_4D_LAYOUT would break this code.